README.md

```diff
@@ -1 +1,55 @@
-# phase_zinc
\ No newline at end of file
+# phase_zinc
+
+## Contents
+*  [Overview](#overview)
+*  [Files included](#files-included)
+*  [How to set up](#how-to-set-up)
+
+## Overview
+A custom phase extension allowing [bazelbuild/rules_scala](https://github.com/bazelbuild/rules_scala) users to use the [Zinc compiler](https://github.com/sbt/zinc). To learn what "customizable phases" are and how the phase architecture can help your Bazel project, see [this doc on customizable phases](https://github.com/bazelbuild/rules_scala/blob/master/docs/customizable_phase.md) on the [bazelbuild/rules_scala](https://github.com/bazelbuild/rules_scala) project.
+
+This project is aimed at anyone who depends on the [bazelbuild/rules_scala](https://github.com/bazelbuild/rules_scala) project but would like to swap out the default Scala compiler with Zinc compiler. This is especially helpful for codebases that have Bazel targets with large numbers of source files-- if Bazel targets are not granular enough, builds can take a long time. Because Zinc is an incremental compiler, it can make builds significantly faster. However, be aware that Zinc can also add non-determinism to your builds.
+
+## Files included
+
+Phase_zinc defines a Zinc compiler phase, then adds this phase to three Bazel rules originally defined in [bazelbuild/rules_scala](https://github.com/bazelbuild/rules_scala): `scala_binary`, `scala_library` and `scala_test`. The new corresponding rules are named, respectively, `zinc_scala_binary`, `zinc_scala_library` and `zinc_scala_test`.
```

Comment:
In `rules/scala.bzl`, we specifically import make functions for `scala_binary`, etc, and then we implement `zinc_scala_binary` in the same file. Are you suggesting we remove that from `scala.bzl`?

Reply:
I drew inspiration from https://github.com/bazelbuild/rules_scala/blob/master/docs/phase_scalafmt.md, but I also noted that `zinc_scala_binary`, etc, are already defined in `scala.bzl` so that's why I wrote it this way. Do you think we should do it more in the style of the the `phase_scalafmt` doc?